docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -305,39 +291,37 @@ spec:
   - openstack-networker
 EOF
 ----
-
-Note: alternatively, you could include the `Networker` specific node set in the
-`nodeSets` list when creating the main, `openstack`,
-`OpenStackDataPlaneDeployment` CR.
-
-Note that once a `OpenStackDataPlaneDeployment` CR is deployed, it's impossible
-to add new node sets to it later.
++
+[NOTE]
+Alternatively, you can include the `Networker` specific node set in the
+`nodeSets` list when you create the main `OpenStackDataPlaneDeployment` CR, before you deploy it. Note that once an `OpenStackDataPlaneDeployment` CR is deployed, you cannot add new node sets to it later.
```

Comment:
"Alternatively, you can include the `Networker` specific node set in the `nodeSets` list before you deploy the main `OpenStackDataPlaneDeployment` CR. You cannot add new node sets to an `OpenStackDataPlaneDeployment` CR after deployment."

Same comment as before re putting "note that" in a [NOTE] block and using "once" instead of "if" or "after" or "when".